lib/manager/batect-wrapper/artifacts.ts

```diff
@@ -0,0 +1,45 @@
+import { logger } from '../../logger';
+import { Http } from '../../util/http';
+import { UpdateArtifact, UpdateArtifactsResult } from '../common';
+
+const http = new Http('batect-wrapper');
+
+async function getFileContent(
+  version: string,
+  fileName: string
+): Promise<string> {
+  const url = `https://github.com/batect/batect/releases/download/${version}/${fileName}`;
+
+  try {
+    const response = await http.get(url);
+
+    return response.body;
+  } catch (err) {
+    const errorDescription: string = err.toString();
```

Comment:
Move try catch to updateArtifacts and return artifacts error instead throw. Checkout other manager like gradle wrapper.